packages/design-system-scripts/gulp/docs/generatePages/uniquePages.js

```diff
@@ -11,8 +11,10 @@ function uniquePages(pages) {
   pages.forEach((page) => {
     if (routes[page.reference]) {
       if (routes[page.reference].source.path.match(/node_modules/)) {
-        // We override pages that come from `node_modules`
+        // We override pages that come from node_modules with pages from the child design system
         logTask('🖊  ', `Overriding ${page.reference || 'index'} page with ${page.source.path}`);
+        // Creates cmsds property used on child design systems doc pages to show a link to the corresponding component from the core CMS Design System
```

Comment:
The `addCmsdsLink()` function in `generatePages` is what actually creates the property, this line is just to make sure that property is preserved when overriding. I would reword to be:

```
// Preserves the `cmsds` property created by `addCmsdsLink()` when overriding
// Even when a page is overridden by a child design system, we want to show the CMSDS link 
```